src/xcode/ENA/ENA/Source/Scenes/Events/TraceLocations/Configuration/TraceLocationConfigurationViewController.swift

```diff
@@ -229,12 +229,32 @@ class TraceLocationConfigurationViewController: UIViewController, FooterViewHand
 	private func temporaryDefaultLengthHeaderTapped() {
 		viewModel.temporaryDefaultLengthHeaderTapped()
 		view.endEditing(true)
+		
+		// If iPhone resolution height is less than or equal to 844 then the screen will auto scroll to see the event duration picker
+		// if the height is more than 844 then user can see the event duration picker without auto scroll.
+		// For screen resolution please see more details on: https://www.ios-resolution.com/
+		if !viewModel.temporaryDefaultLengthPickerIsHidden && UIScreen.main.bounds.size.height <= 667 {
```

Comment:
Your solution is fixing the bug.

From the code base, I would prefer a common Helper class for device bounds (device resolutions).
Similar to that https://github.com/anatoliyv/AssistantKit

But at the moment your solution is fine for me